src/eureka/S3_data_reduction/plots_s3.py

```diff
@@ -28,8 +28,8 @@ def lc_nodriftcorr(meta, wave_1d, optspec, optmask=None):
     None
     '''
     normspec = util.normalize_spectrum(meta, optspec, optmask=optmask)
-    wmin = np.ma.min(wave_1d)
-    wmax = np.ma.max(wave_1d)
+    wmin = np.nanmin(wave_1d)
+    wmax = np.nanmax(wave_1d)
```

Comment:
Did you mean to do this? I guess wave_1d isn't a masked array so you needed to use the np.nan versions to handle those strange values you mentioned in your issue?